lib/projetopay/account.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Projetopay.Account do
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+   alias Projetopay.User
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+
+  @required_params [:balance, :user_id]
+
+  schema "accounts" do
+    field :balance, :decimal
```

Comment:
Depois de alterar no banco, tem que alterar aqui também para `:integer`

```suggestion
    field :balance, :integer
```